### PR TITLE
VS: scrape daily PDF

### PR DESCRIPTION
**Pull request by maekke**
the [latest PDF](https://www.vs.ch/documents/6756452/7008787/2020%2010%2030%20-%20Sit%20Epid%20-%20%C3%89tat%20Stand) contains more recent data than the excel file, might be worth scraping this file...

### CODE REVIEW

File: .github/workflows/run_scrapers.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
         pip install -r requirements.txt
         sudo apt update || true # do not fail if update does not work
         sudo apt-get install sqlite3
-        if [ "$SCRAPER_KEY" = "FL" ] || [ "$SCRAPER_KEY" = "GE" ] || [ "$SCRAPER_KEY" = "VD" ] || [ "$SCRAPER_KEY" = "GL" ] || [ "$SCRAPER_KEY" = "SZ" ] || [ "$SCRAPER_KEY" = "TI" ] || [ "$SCRAPER_KEY" = "SO" ] ; then
+        if [ "$SCRAPER_KEY" = "FL" ] || [ "$SCRAPER_KEY" = "GE" ] || [ "$SCRAPER_KEY" = "VD" ] || [ "$SCRAPER_KEY" = "GL" ] || [ "$SCRAPER_KEY" = "SZ" ] || [ "$SCRAPER_KEY" = "TI" ] || [ "$SCRAPER_KEY" = "SO" ] || [ "$SCRAPER_KEY" = "VS" ] ; then
```
Comment (metaodi) [inline .github/workflows/run_scrapers.yml:64]: It's probay time to get rid of this condition andmsimply install it for all cantons.

It made sense when only a couple of cantons needed PDF scraping capabilities.

File: scrapers/scrape_vs.py [hunk 2]
```diff
@@ -37,5 +66,4 @@
         dd.recovered = sum(r['Nb de nouvelles sorties'] for r in rows[:i+1])
     if not is_first:
         print('-' * 10)
-    is_first = False
```
Comment (metaodi) [inline scrapers/scrape_vs.py:40]: I think this is still needed, should the condition above (line 35) be false, then `is_first` must be set here.

Comment (maekke) [reply scrapers/scrape_vs.py]: Whoops, you're right! Reverted this change in d9dcf5c

File: .github/workflows/run_scrapers.yml (hunk 1 above)
Comment (maekke) [reply .github/workflows/run_scrapers.yml]: I thought about that as well, should be addressed in 77f43e3.
Or should we just have one `sudo apt-get install xxx` call with sqlite3 and poppler-utils in one shot?

**Pull request merged**